opencti-platform/opencti-graphql/src/domain/settings.js

```diff
@@ -136,6 +136,8 @@ export const getSettings = async (context) => {
     platform_ai_type: `${getAIEndpointType()} ${nconf.get('ai:type')}`,
     platform_ai_model: nconf.get('ai:model'),
     platform_ai_has_token: !!isNotEmptyField(nconf.get('ai:token')),
+    platform_trash_enabled: nconf.get('app:trash:enabled') ?? true,
+    platform_trash_retention_days: nconf.get('garbage_collection_manager:deleted_retention_days'),
```

Comment:
I'm not sure if this will work on multi-node config: we would ideally like to return the lowest retention days of all running garbage collection managers?

Reply:
I will remove the mention, it's too complex for now to consolidate all managers configurations from other nodes.